genkit-tools/cli/src/commands/start.ts

```diff
@@ -76,12 +48,53 @@ export const start = new Command('start')
       } else {
         port = await getPort({ port: makeRange(4000, 4099) });
       }
-      uiPromise = startManager(true).then((manager) =>
-        startServer(manager, port)
-      );
+      uiPromise = startManager(true).then((manager) => {
```

Comment:
wait... you still need the manager in noui mode?

Reply:
This is in ui mode

https://github.com/firebase/genkit/blob/bad32552acd65b344334c2dde59f7b62693c961b/genkit-tools/cli/src/commands/start.ts#L40

Reply:
Yes. We need the manager in both modes.